src/server/parameters.js

```diff
@@ -1,5 +1,5 @@
-var answerTime = 5; //answer time length in seconds
-var submitTime = 5;
+var answerTime = 30; //answer time length in seconds
```

Comment:
Handy dev tip: prefer longer names that explain variables over adding comments - keeping comments up to date can cause developers in the future problems.